samples/15.c.programmatic-post-activity/README.md

```diff
@@ -0,0 +1,21 @@
+# Sample - Programmatic access to post activity
+
+When a button is clicked, it will programmatically send a message to the bot.
+
+# Test out the hosted sample
+
+- [Try out MockBot](https://microsoft.github.io/BotFramework-WebChat/15.c.programmatic-post-activity)
+
+# Things to try out
+
+- Click the "Help" button on upper-left hand corner
+
+# Further reading
+
+## Related samples
+
+[`15.b.incoming-activity-event`](https://github.com/Microsoft/BotFramework-WebChat/tree/master/samples/15.a.incoming-activity-event) is a sample which will fire JavaScript event on all incoming activity.
```

Comment:
a sample ***that*** will fire JavaScript events on all incoming ***activities***